ChatApp/View/Sender/SenderCapsuleBubble.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  SenderCapsuleBubble.swift
+//  ChatApp
+//
+//  Created by Ana Dzebniauri on 25.04.23.
+//
+
+import UIKit
+
+class SenderCapsuleBubble: UIView {
+    
+    private var radius: CGFloat {
+        bounds.height/2
```

Comment:
same here as above you should use radius from design and have dynamic radius is not correct